requirements.txt

```diff
@@ -0,0 +1,26 @@
+# Dev packages
+--editable git+https://github.com/inducer/pymbolic.git#egg=pymbolic
+#--editable git+https://github.com/inducer/islpy.git#egg=islpy
+#--editable git+https://github.com/inducer/pyopencl.git#egg=pyopencl
```

Comment:
Why are these commented out? *And* removed by `grep` above?

Reply:
Better safe than sorry :-) Fixed in d26e39e (the commented part)